src/url.cpp

```diff
@@ -298,4 +305,76 @@ namespace Url
         return *this;
     }
 
+    Url& Url::relative(const std::string& other)
```

Comment:
I would support as an overload a version that takes a `Url` object as an `other` to avoid the inline parse.